Cron/Manager.php

```diff
@@ -12,23 +12,23 @@
 use Cron\CronBundle\Entity\CronJob;
 use Cron\CronBundle\Entity\CronJobRepository;
 use Cron\CronBundle\Entity\CronReport;
+use Doctrine\Common\Persistence\ManagerRegistry;
```

Comment:
`Doctrine\Common\Persistence` was moved to `Doctrine\Persistence`

Reply:
A much seems to be changing at doctrine at the moment. Well, I have solved that. Thanks for the review.